src/components/common/styles/Resources.css

```diff
@@ -28,3 +28,18 @@
   justify-content: space-between;
   padding: 4% 0;
 }
+
+[title="Resource"] {
+  color: black !important;
```

Comment:
I agree with Ryan--who commented earlier. I think there's a better way to set this CSS than using `!important`. It's definitely a cleaner result, but I think using this solution will lead to problems down the road. I would recommend inline styling or playing with this CSS file to get it to stick without using !important.

Reply:
Fixed these changes you requested!

Reply:
I like this a lot. :)
Don't know if you need to record a new loom video to show these new changes or not, but I think this approach is much more manageable down the road.